stdlib/2/subprocess.pyi

```diff
@@ -70,33 +78,31 @@ class Popen:
     returncode = 0
 
     def __init__(self,
-                 args: Union[str, Sequence[str]],
+                 args: _CMD,
                  bufsize: int = ...,
-                 executable: Optional[str] = ...,
+                 executable: Optional[_TXT] = ...,
                  stdin: Optional[_FILE] = ...,
                  stdout: Optional[_FILE] = ...,
                  stderr: Optional[_FILE] = ...,
                  preexec_fn: Optional[Callable[[], Any]] = ...,
                  close_fds: bool = ...,
                  shell: bool = ...,
-                 cwd: Optional[str] = ...,
-                 env: Optional[Mapping[str, str]] = ...,
+                 cwd: Optional[_TXT] = ...,
+                 env: Optional[Mapping[_TXT, _TXT]] = ...,
                  universal_newlines: bool = ...,
                  startupinfo: Optional[Any] = ...,
                  creationflags: int = ...) -> None: ...
 
     def poll(self) -> int: ...
     def wait(self) -> int: ...
-    def communicate(self, input: Optional[AnyStr] = ...) -> Tuple[Optional[bytes], Optional[bytes]]: ...
+    # morally: -> Tuple[_TXT, _TXT]
+    def communicate(self, input: Optional[_TXT] = ...) -> Tuple[Any, Any]: ...
```

Comment:
I think `Any` is preferable in the same way it is for the other sort of `output` annotations; with `Optional`, a user who passed in a `PIPE` must explicitly check for None (or cast). I'm updating the descriptive comment to say the right thing though (the original annotation).